bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_Dilepton_NLO/HeavyNeutrinoToDilepton_Schannel_NLO/HeavyNeutrinoToEMu_Schannel_M400_NLO/HeavyNeutrinoToEMu_Schannel_M400_NLO_run_card.dat

```diff
@@ -0,0 +1,74 @@
+  tag_1     = run_tag ! name of the run 
```

Comment:
Hi, sorry for the neverending comments, but here I feel like you removed too much...
The run_card has several commented lines which contain useful comments and descriptions,
these lines were not meant to be removed...

Reply:
Hi, you don't have to be sorry, apologies should be from my side :(. I was not sure which you were talking about from the last comment. Should I use $DEFAULT_PDFSETS instead of specifying one?

Reply:
Yes, 
$DEFAULT_PDF_SETS = lhaid
$DEFAULT_PDF_MEMBERS = reweight_PDF